01_async.js

```diff
@@ -0,0 +1,51 @@
+const request = require('request-promise');
+
+console.log('start ...');
+
+/**
+ * Get repos with their contributors 
+ * @param {string} organization 
+ * @param {int} limit 
+ */
+async function getContributors(organization, limit = 3) {
+    async function getContributorsOfReposByOwner(owner, repo) {
```

Comment:
async function everytime returns promise. Here await unpacks promise, but async packs value in promise.
More efficient `return request`